docs/source/getting_started.rst

```diff
@@ -957,60 +942,67 @@ alpha.
 This ParametricProgram now acts as a template, caching the result of the ``exponential_map``
 calculation so that it can be used later with new values.
 
-JobConnections
---------------
-Larger pyQuil programs can take longer than 5 seconds to run.  These jobs can be posted into the
-cloud job queue using a different connection object.  The mode of interaction with the API is
-asynchronous.  This means that there is a seperate query to post a job and to get the result.
+Connections
+-----------
+Larger pyQuil programs can involve more qubits and take a longer time to run. Instead of running the
+program immediately, you can insert your programs into a queue. This is done with the ``skip_queue``
```

Comment:
I think it would be conceptually simpler to negate this, i.e.  This is done with the `use_queue` parameter to... By default this is set to False...